tools/savefileimporter/code/character_parser.dm

```diff
@@ -0,0 +1,207 @@
+#define READ_FILE(sf, varname, fallback) var/##varname; sf >> ##varname; if(!##varname) ##varname=fallback;
+#define READ_FILE_EXVAR(sf, varname) sf >> ##varname;
+
+var/global/list/custom_name_types = list(
+	"human",
+	"clown",
+	"mime",
+	"cyborg",
+	"ai",
+	"religion",
+	"deity",
+)
+
+// In its own file because its such a mess
+/proc/parse_characters(owning_ckey, savefile/S, list/cdirs)
+	for(var/character_dir in cdirs)
+		S.cd = character_dir
```

Comment:
I think this needs a / before it?
![image](https://user-images.githubusercontent.com/26465327/191816480-8fe73e17-aa96-4ca3-ae9b-65828eb036b7.png)

Judging by all current implementations of this

Reply:
It appeared to import in my testing, but I will need to look into *why* its scuffed.

Not being familiar with this codebase at all probably doesn't help matters,

Reply:
It imported the first 1, but character2 etc. failed

Reply:
Tested with Bacon and confirmed that it does indeed need to be 
```suggestion
		S.cd = "/[character_dir]"
```